docs/reference/c++_support.rst

```diff
@@ -0,0 +1,165 @@
+.. meta::
+  :description: This chapter describes the C++ support of the HIP ecosystem
+  ROCm software.
+  :keywords: AMD, ROCm, HIP, C++
+
+*******************************************************************************
+C++ language support
+*******************************************************************************
+
+.. _language_introduction:
+Introduction
+===============================================================================
+
+The main way to harness the power of the ROCm platform is through the usage C++ and HIP
+code. This code is then compiled with a ``clang`` or ``clang++`` compiler. The official
+versions support the HIP platform, but for the most up-to-date feature set use the
+``amdclang`` or ``amdclang++`` installed with your ROCm installation.
```

Comment:
The ROCm platform enables the power of combined C++ and HIP code. This code is compiled with a ``clang`` or ``clang++`` compiler. The official compilers support the HIP platform, but for the most up-to-date feature set, use the ``amdclang`` or ``amdclang+``+ components included in the ROCm installation.